### ISSUE TITLE
Console output interpretation

### ISSUE THREAD

**Issue by fsmosca** [glinscott/nnue-pytorch#83]
### Input data
Training pos: 500m depth 5
Validation pos: 1m depth 10

### Command line
```
python train.py --flush_logs_every_n_steps 1 --max_epochs 200 --smart-fen-skipping --batch-size 8192 --threads 4 --num-workers 4 --gpu 1 ncsf_2021-02_pos500000000_d5_train_tn20.binpack ncsf_2021-02_pos1000000_d10_val_tn10.binpack
```

### Console output
```
Epoch 1:  39%|█████████▍              | 4846/12331 [05:59<09:15, 13.47it/s, loss=0.0136, v_num=1]
```

What are the following?
`4846/12331 [05:59<09:15, 13.47it/s, loss=0.0136, v_num=1]`

**Comment by Sopel97** [comment]
`step in the current epoch`/`total steps in the current epoch` [`time elapsed in the current epoch`<`time left in the current epoch`, `number of batches per second`it/s, loss=`training loss (cross entropy)`, v_num=`version, automatically detected such that logs from different runs are in different dirs`]

**Comment by fsmosca** [comment]
Thanks.
